tutorials/interface/activity1.md

```diff
@@ -0,0 +1,242 @@
+# Get to Know MakeCode Arcade
+
+### @explicitHints true
+
+## Introduction @unplugged
+
+![Psyched Monkey](https://github.com/microsoft/pxt-arcade/blob/master/docs/static/tutorials/monkey.png?raw=true "Psyched Monkey is Ready!" )

```

Comment:
@shakao  @riknoll what's the story for image in skill map context? Pulling files from github won't work.

Reply:
hmm haven't thought too much about this but if you publish on github pages it hosts all your content (that's how we do the the certificates right now), so something like `https://microsoft.github.io/pxt-skillmap-sample/img/space/activity1.png` would work for the images also

Reply:
If github pages are enabled, this will work

https://microsoft.github.io/pxt-skillmap-sample/img/platformer/activity2.png

Reply:
Talked to shakao on this and should be able to use a relative link if I keep putting images in the pxt-arcade tutorials folder.